CovidSafe/CovidSafe.DAL/Repositories/Cosmos/CosmosMatchMessageRepository.cs

```diff
@@ -86,6 +96,7 @@ public async Task<IEnumerable<MessageInfo>> GetLatestAsync(Region region, long l
             var iterator = queryable
                 .Where(r =>
                     r.Timestamp > lastTimestamp
+                    && r.Timestamp >= this._getMaxDataAge()
```

Comment:
[nit] maybe something like timestampThreshold = Max(lastTimestamp, getMaxDataAge()) and single condition in query is abit cleaner?

Reply:
I like this idea much better.